index.js

```diff
@@ -98,6 +98,13 @@ CanvasRenderingContext2D.prototype.beginCircle = function (x, y, r) {
   this.arc(x, y, r, 0, 2.0 * Math.PI);
 };
 
+//create a String containing all the possible shortcodes for the shapes
```

Comment:
It is apparent from the code and variable names what it is happening. You don't need to restate it as comments.